roles/pulp3-postgresql/tasks/main.yml

```diff
@@ -12,6 +12,14 @@
         state: present
       when: ansible_distribution == 'CentOS'
 
+      # Workaround geerlingguy issue
+      # https://github.com/geerlingguy/ansible-role-postgresql/issues/48
+    - name: Install psycopg2 package
+      package:
+        name: 'python2-psycopg2'
+        state: present
+      when: ansible_distribution == 'Fedora'
```

Comment:
I can add this as part of my open PR to upstream role:

https://github.com/geerlingguy/ansible-role-postgresql/pull/53

Reply:
Awesome! Thank you for contributing upstream. After that is merged, if you're able to, maybe remove this from here when its no longer needed?